src/requirements.txt

```diff
@@ -81,6 +81,7 @@ knack==0.8.0rc2
 matplotlib
 matplotlib-inline
 mistune==0.8.4
+mmocr
```

Comment:
In the long run I think we'll need to start splitting requirements per sub-project, but we'll deal with that later.